#goal definition
string command
float32 layer_dist
---
#result definition
string result
---
#feedback
string feedback
